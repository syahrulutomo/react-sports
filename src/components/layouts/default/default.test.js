import React from 'react';
import renderer from 'react-test-renderer';
import { LayoutDefault } from '.';

it('renders without crashing', () => {
  const component = renderer.create(<LayoutDefault />);

  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
