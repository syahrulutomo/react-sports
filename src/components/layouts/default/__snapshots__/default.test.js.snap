// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<main
  id="app-layout--default"
/>
`;
